requirements.txt

```diff
@@ -1,2 +1,7 @@
+certifi==2020.12.5
+chardet==4.0.0
+idna==2.10
 python-dateutil==2.8.1
 requests==2.25.1
+six==1.15.0
+urllib3==1.26.3
```

Comment:
Why should we list these modules? Can't we leave the dependancies to the imported modules?

Reply:
It has been best practice to specify all dependencies and their dependencies in the most explicit manner possible. This gives a really well defined development environment and prevents incompatibilities / inconsistencies between dev setups.

However, I'm adaptive, if you like to maintain with minimal requirements / most flexibility.